src/test/java/com/example/rets_api/utils/DtoUtilsTest.java

```diff
@@ -41,6 +41,29 @@ public static PropertyDTO createPropertyDTOWithBasicFields(){
             .build();
     }
 
+    public static PropertyPatchDTO createPropertyPatchDTOWithBasicFields(){
```

Comment:
Please change the name to createPropertyPatchDTO, because will have just basics fields by default ;)